### PR TITLE
Use dtk-template for ninja build

### PR DESCRIPTION
**Pull request by encounter**
Thanks @roeming and @EpochFlame for helping push this along

### CODE REVIEW

**Review by EpochFlame — DISMISSED** [review]
lgtm. any objections?

File: src/Dolphin/MSL_C/MSL_Common/FILE_POS.c
Comment (EpochFlame) [inline src/Dolphin/MSL_C/MSL_Common/FILE_POS.c:1]: reasonably sure the original file had the capitalized extension- check map

**Review by roeming — DISMISSED** [review]

**Review by EpochFlame — APPROVED** [review]
cool

File: include/PSSystem/FxMgr.h
[the remark this answers is not in the document]
Comment (EpochFlame) [reply include/PSSystem/FxMgr.h:1]: it was indeed vestigial

**Review by EpochFlame — APPROVED** [review]
we’re good

**Pull request merged**